arrays/Leetcode/remove_element.py

```diff
@@ -0,0 +1,27 @@
+def remove_element(nums, val):
+    k = 0
+    lens = 0
+    # create an empty array
+    vals = []
+
+    for i in range(len(nums)):
+        if nums[i-k] == val:
+            nums.append(val)
+            nums.pop(i-k)
+            k += 1
+        else:
+            lens += 1
+    return lens
+
+def test(did_pass):
+    if did_pass:
+        res = "test ok"
+    else:
+        res = "test failed"
+    print(res)
+
+def test_suite():
+    test(remove_element([3,2,2,3], 3) == 2)
+
+test_suite()
+
```

Comment:
- basically initialize a value, k to 0 and your lengths, lens to 0
- create an empty array vals
- while looping, check if the index - k is equal to the val
- if true append val to our nums array
- proceed to pop the value that is equal to val
- increase counter on k
- else increase count on our lengths, lens